notification/slack.py

```diff
@@ -166,13 +167,28 @@
 OLD_SLACK_INCOMING_WEBHOOK = 'https://%s/services/hooks/incoming-webhook?token=%s'
 SLACK_INCOMING_WEBHOOK = 'https://hooks.slack.com/services/%s'
 
+import urllib
```

Comment:
At first glance I'm not sure if this `urllib` import is necessary... am I wrong? If so please ignore.